vips/vip-405/bscmainnet.ts

```diff
@@ -0,0 +1,170 @@
+      },
+      {
+        target: OPMAINNET_ACM_AGGREGATOR,
+        signature: "executeRevokePermissions(uint256)",
+        params: [0],
+        dstChainId: LzChainId.opmainnet,
+      },
+      {
+        target: OPMAINNET_ACM,
+        signature: "revokeRole(bytes32,address)",
+        params: [DEFAULT_ADMIN_ROLE, OPMAINNET_ACM_AGGREGATOR],
+        dstChainId: LzChainId.opmainnet,
+      },
+
+      {
+        target: ETHEREUM_ACM,
+        signature: "grantRole(bytes32,address)",
+        params: [DEFAULT_ADMIN_ROLE, ETHEREUM_ACM_AGGREGATOR],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ETHEREUM_ACM_AGGREGATOR,
+        signature: "executeGrantPermissions(uint256)",
+        params: [2],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ETHEREUM_ACM_AGGREGATOR,
+        signature: "executeRevokePermissions(uint256)",
+        params: [1],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ETHEREUM_ACM,
+        signature: "revokeRole(bytes32,address)",
+        params: [DEFAULT_ADMIN_ROLE, ETHEREUM_ACM_AGGREGATOR],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: OPBNB_ACM,
+        signature: "grantRole(bytes32,address)",
+        params: [DEFAULT_ADMIN_ROLE, OPBNB_ACM_AGGREGATOR],
+        dstChainId: LzChainId.opbnbmainnet,
+      },
+      {
+        target: OPBNB_ACM_AGGREGATOR,
+        signature: "executeRevokePermissions(uint256)",
+        params: [1],
+        dstChainId: LzChainId.opbnbmainnet,
+      },
+      {
+        target: OPBNB_ACM,
+        signature: "revokeRole(bytes32,address)",
+        params: [DEFAULT_ADMIN_ROLE, OPBNB_ACM_AGGREGATOR],
+        dstChainId: LzChainId.opbnbmainnet,
+      },
+
+      {
+        target: ARBITRUM_ACM,
+        signature: "grantRole(bytes32,address)",
+        params: [DEFAULT_ADMIN_ROLE, ARBITRUM_ACM_AGGREGATOR],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_ACM_AGGREGATOR,
+        signature: "executeGrantPermissions(uint256)",
+        params: [1],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_ACM_AGGREGATOR,
+        signature: "executeRevokePermissions(uint256)",
+        params: [1],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_ACM,
+        signature: "revokeRole(bytes32,address)",
+        params: [DEFAULT_ADMIN_ROLE, ARBITRUM_ACM_AGGREGATOR],
+        dstChainId: LzChainId.arbitrumone,
+      },
+
+      {
+        target: BSC_ACM,
+        signature: "grantRole(bytes32,address)",
+        params: [DEFAULT_ADMIN_ROLE, BSC_ACM_AGGREGATOR],
+      },
+      {
+        target: BSC_ACM_AGGREGATOR,
+        signature: "executeGrantPermissions(uint256)",
+        params: [0],
+      },
+      {
+        target: BSC_ACM,
+        signature: "revokeRole(bytes32,address)",
+        params: [DEFAULT_ADMIN_ROLE, BSC_ACM_AGGREGATOR],
+      },
```

Comment:
I think I prefer to continue using single commands for the management of permissions on BNB Chain. It's totally fine for me to use the aggregator here, because the commands are already preloaded